.github/workflows/pull_request.yml

```diff
@@ -65,6 +67,8 @@ jobs:
         with:
           run: sh -c "find . -mindepth 1 -delete"
       - uses: actions/checkout@v2
+      - name: Check License Header
+        uses: apache/skywalking-eyes@main
```

Comment:
This job no longer needs to check the license repeatedly, because the `lint` job has already checked it.